platforms/android/include.gradle

```diff
@@ -38,3 +38,5 @@ dependencies {
 //    compile "com.google.android.gms:play-services-auth:9.6.+"
 
 }
+
+apply plugin: "com.google.gms.google-services"
```

Comment:
Indeed, adding `apply plugin: "com.google.gms.google-services"` to the bottom is no longer required. Cool!

I see what you mean with the after prepare hook but I think it's worth the brittleness to include it in the plugin. Worst case scenario the developer still needs to add the classpath by hand but generally this would make installation lots easier.